### DISCUSSION TITLE
MenuItem needs to scroll

### DISCUSSION THREAD

**Discussion by rseydler** [iTwin/iTwinUI-react#215] [Q&A] [answered]
I have a DropdownMenu with approx 30 items in it from client data. This amount of data has the list ending off screen :( Is there a way to handle scrolling in the dropdown itself?

Thanks!

**Comment by veekeys** [comment, marked as answer]
Hello @rseydler,

This is achievable with simple css as you would do it to any other HTML element to make it scrollable.
Just apply class on the `DropdownMenu` component with `overflow-y: auto` and `max-height` to whatever size you want.
Check up this code sandbox for an example:
https://codesandbox.io/s/itwinui-react-scrollable-dropdown-y5ubb?file=/src/App.tsx

**Reply by rseydler** [reply]
Thanks for taking the time to reply to my beginner question. I need to stop thinking like a winforms developer!

**Reply by mayank99** [reply]
Just fyi, it is better to use `overflow-y: overlay` (and fallback to `auto`). Overlay scrollbars will appear over the content, and usually look better.

| auto | overlay |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/9084735/126792749-138d547e-60f7-4391-8827-ecb8c6a4a402.png) | ![image](https://user-images.githubusercontent.com/9084735/126792834-0f14859f-2752-4046-ab6d-b08de13c13a0.png) |

**Reply by veekeys** [reply]
@rseydler I have updated the sandbox with @mayank99 suggestion, so you could check it up and apply same in your project.
